src/components/AppSidebarTabs/index.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * @copyright 2019 Christoph Wurst <christoph@winzerhof-wurst.at>
+ *
+ * @author 2019 Christoph Wurst <christoph@winzerhof-wurst.at>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import AppSidebarTabs from './AppSidebarTabs'
+
+export default AppSidebarTabs
```

Comment:
I'm wondering if this shouldn't just be next to the AppSidebar.vue as thi sis a non-exported component?

Reply:
You mean to not export `AppSidebarTabs`, but directly import it in `AppSidebar`? Probably makes sense. I will adjust it in the evening.

In the future, we could in principle also export `AppSidebarTabs` (maybe export it as `Tabs`). The component should work without `AppSidebar` as well.

Reply:
Done.